src/event-notion/index.ts

```diff
@@ -343,6 +344,7 @@ export const pingForTAPandCSIDeadlines = async (
     message: string;
     prop: string;
     propStatus: string[];
+    pingHosts: boolean,
```

Comment:
Could just make it 
```ts
pingHosts ?: boolean
```
so you don't have to specify `pingHosts: false` everywhere